content/en/blog/_posts/2023-11-16-mid-cycle-1.29.md

```diff
@@ -28,6 +28,9 @@ To host its container images, the Kubernetes project uses a community-owned imag
 
 ## Deprecations and Removals for Kubernetes v1.29
 
+Follow-up the official list of [API removals](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29) planned for Kubernetes v1.29.
```

Comment:
```suggestion
Follow-up the official list of [API removals](/docs/reference/using-api/deprecation-guide/#v1-29) planned for Kubernetes v1.29.
```

Reply:
Change to relative link.

Reply:
@kcmartin  please review this commit I fixed to use relative link, [here](https://github.com/kubernetes/website/pull/43439/commits/e4bad64f1d9f52be9dc2b06f4415926f49bc80ee)